app/utils/UtilFunc.js

```diff
@@ -1,24 +1,33 @@
 export const isFunc = functionToCheck =>
   functionToCheck && {}.toString.call(functionToCheck) === '[object Function]';
 
-export const formatBalance = balance =>
-  parseFloat(balance)
+export const formatBalance = balance => {
+  if (
+    parseFloat(balance) > 0 &&
+    Number(balance) === balance &&
```

Comment:
I tried it and worked quite well, can you give a digit that you found that it does work for

Reply:
Okay you can take a look at the code here : https://replit.com/@kayodeokunlade/PleasedAutomaticFormula#index.js

The if block never gets executed for values passed only the last part gets executed which rounds the value up.
```
formatBalance("0.993990") is rounded up to 0.9940 because the if block never gets executed due to the conditions

```